### PR TITLE
FEAT: make 1D histograms interactive with sliders

### PR DESCRIPTION
**Pull request by shenvitor**
First step towards #31. This only makes the 1D histograms interactive, by creating sliders for the width and mass of each resonance.

### CODE REVIEW

**Comment by redeboer** [comment]
Good to have a dedicated playground notebook!
Eventually I would remove it, but it is useful to try the mechanism with a small example. The examples you give will have to be modified though: the plot has to be updated rather than redrawn, so we need to define the figure outside the function and define some global state for the line that is to be ploted. See the examples on ComPWA.

**Comment by shenvitor** [comment]
Collecting some reference tutorials:
https://kapernikov.com/ipywidgets-with-matplotlib/

**Comment by redeboer** [comment]
Seems that the code in that tutorial is not the most efficient — it removes the plot and then creates a new one (still; better than removing the entire figure).

You may also consider using [`mpl-interactions`](https://mpl-interactions.readthedocs.io). This has good (and fast) support for matplotlib's core plotting objects, but you loose flexibility, e.g. you can't quickly draw an additional line that updates along with your parameters.

An example of an interactive histogram can be found here:
https://compwa.github.io/ampform-dpd/comparison/d2kkk.html
Note how the final cell defines a global state `lines`, which is a list of `step` lines that are generated `if lines is None` and that are updated with `set_ydata()` upon changes to the sliders (i.e. the parameter values). The code isn't pretty, but it is flexible enough to extend with additional indicators on the plot.

**Review by redeboer — CHANGES_REQUESTED** [review]

File: docs/ampform/LambdaKpi0.ipynb [hunk 1]
```diff
@@ -533,13 +533,13 @@
...
     "\n",
-    "tabs = []\n",
-    "for category in categorized_sliders_m:\n",
-    "    box_m = w.VBox(categorized_sliders_m[category])\n",
-    "    box_gamma = w.VBox(categorized_sliders_gamma[category])\n",
-    "    combined_box = w.HBox([box_m, box_gamma])\n",
-    "    tabs.append(combined_box)\n",
-    "\n",
+    "tabs = [\n",
+    "    w.HBox([\n",
```
Comment (redeboer) [inline docs/ampform/LambdaKpi0.ipynb:540]: @shenvitor this is what I meant: the 'hugging' of `([` in the `HBox` constructor. That only works if preview for Ruff is activated for the formatter (not just the linter), see 66a312d.

**Review by redeboer — APPROVED** [review]
Awesome 🌵

**Pull request merged**